### PR TITLE
remove replaced endpoints /api/seal, /api/encode, /api/decode

### PR DESCRIPTION
**Pull request by adrianiacobghiula**

### CODE REVIEW

**Comment by bakito** [comment]
@adrianiacobghiula can you please check the e2e tests, as tey currently fail.

**Comment by adrianiacobghiula** [comment]
Could you please release a new version after the merge of this PR (and maybe a release before the merge of this PR)

File: pkg/handler/dencode.go [hunk 1]
```diff
@@ -1,126 +1,77 @@
 package handler
```
Comment (bakito) [inline pkg/handler/dencode.go:1]: Could you please write here some unit tests for functions dencode and readSecret?

Comment (adrianiacobghiula) [reply pkg/handler/dencode.go:1]: sure sure

**Comment by bakito** [comment]
> Could you please release a new version after the merge of this PR (and maybe a release before the merge of this PR)

I made a pre release [v3.0.0-pre](https://github.com/bakito/sealed-secrets-web/releases/tag/v3.0.0-pre)

As there are anyway many changes in configuration, since I changed to use the sealed-secret go api in stead of using the kubeseal cli, I think it is OK just do make a proper v3.0.0 release with all final changes.

**Pull request merged**